### PR TITLE
common-adapters (R-Z) dark mode support

### PR DESCRIPTION
**Pull request by hyperobject**
You know the drill at this point, mostly changes to `styleSheetCreate` calls so they use closures, refactors to `Styles` references, and now refactors to add a `Kb` object so our common-adapter references stay consistent across the app.

@keybase/react-hackers @keybase/hotpotatosquad

### CODE REVIEW

File: shared/common-adapters/markdown/service-decoration.tsx [hunk 1]
```diff
@@ -31,7 +31,10 @@ type KeybaseLinkProps = {
 
 const KeybaseLink = (props: KeybaseLinkProps) => {
   const dispatch = Container.useDispatch()
-  const onClick = React.useCallback(() => dispatch(DeeplinksGen.createLink({link: props.link})), [dispatch])
+  const onClick = React.useCallback(() => dispatch(DeeplinksGen.createLink({link: props.link})), [
```
Comment (jakob223) [inline shared/common-adapters/markdown/service-decoration.tsx]: good catch :O

File: shared/common-adapters/radio-button.desktop.tsx [hunk 2]
```diff
@@ -18,45 +22,46 @@ const StyledRadio = styled.div(
...
+    border: `solid 1px ${Styles.globalColors.black_20}`,
     opacity: disabled ? 0.4 : 1,
   })
 )
 
 const RadioButton = ({disabled, label, onSelect, selected, style}: Props) => (
   <div
     // @ts-ignore clash between StylesCrossPlatform and React.CSSProperties
-    style={{...styleContainer, ...(disabled ? {} : desktopStyles.clickable), ...style}}
+    style={{...styles.container, ...(disabled ? {} : Styles.desktopStyles.clickable), ...style}}
```
Comment (jakob223) [inline shared/common-adapters/radio-button.desktop.tsx]: is this going to cause problems when people pass in styles in `style` that conflict with `styles.container`?

File: shared/common-adapters/radio-button.native.tsx [hunk 3]
```diff
@@ -39,24 +44,26 @@ const RadioInnerCircle = Styles.styled<typeof ClickableBox, ExtraProps>(Clickabl
 )
 
 const RadioButton = ({disabled, label, onSelect, selected, style}: Props) => (
-  <ClickableBox
-    style={{...styleContainer, ...style}}
+  <Kb.ClickableBox
+    style={{...styles.container, ...style}}
```
Comment (jakob223) [inline shared/common-adapters/radio-button.native.tsx]: same question as above

File: shared/common-adapters/tab-bar.native.tsx [hunk 4]
```diff
@@ -149,27 +162,27 @@ const styleBadgeNav = {
 }
 
 const stylesContainer = {
```
Comment (jakob223) [inline shared/common-adapters/tab-bar.native.tsx]: should these guys go in a stylesheet create?

File: shared/common-adapters/radio-button.desktop.tsx (hunk 2 above)
Comment (chrisnojima) [reply shared/common-adapters/radio-button.desktop.tsx]: no, it overwrites it

Comment (chrisnojima) [reply shared/common-adapters/radio-button.desktop.tsx]: we also do this everywhere. it has a base style and you can override it

File: shared/common-adapters/tab-bar.native.tsx (hunk 4 above)
Comment (hyperobject) [reply shared/common-adapters/tab-bar.native.tsx]: Oh good catch, thanks!

**Pull request merged**